modules/ROOT/pages/tools/neo4j-admin/unbind.adoc

```diff
@@ -5,6 +5,16 @@
 
 You can use the `neo4j-admin server unbind` command to remove and archive the cluster state of a cluster server so that it can rebind to a cluster.
 
+[WARNING]
+====
+Avoid running the `unbind` command on all servers, as this results in the loss of cluster quorum.
```

Comment:
We can loose quorum even when running unbind on any amount of servers depending on the setup the customer is using. Basically unbind is force removing a server which has no respect for minimum cluster size etc.

Perhaps this can warn that running unbind always has a chance of quorum loss

Reply:
I was under impression that we need directly warn against running this command on all servers as users do it trying to fix their clusters.

Reply:
I rewrote the warning. I hope it will help users understand differences between the `unbind` command in 4.4 and in 5.x.